webots_ros2_driver/webots_ros2_driver/ros2_supervisor.py

```diff
@@ -65,6 +65,9 @@ def __init__(self):
         # Services
         self.create_service(SpawnUrdfRobot, 'spawn_urdf_robot', self.__spawn_urdf_robot_callback)
         self.create_service(SpawnNodeFromString, 'spawn_node_from_string', self.__spawn_node_from_string_callback)
+        self.create_service(SetString, 'animation_start_recording', self.__animation_start_recording_callback)
```

Comment:
The services are not namespaced, just like the other services already available in this class. According to the documentation, they should actually be `/supervisor/animation_start_recording`. Should I change it?

Reply:
We should add the namespace to the creation of the `Ros2Supervisor` node. You can add the namespace argument [here](https://github.com/nikolaseu/webots_ros2/blob/enh-record-animation-services/webots_ros2_driver/webots_ros2_driver/webots_launcher.py#L253-L266).

We can use Ros2Supervisor to be as clear as possible which node declared these services.

Reply:
Yes exactly!